CHANGELOG.md

```diff
@@ -64,6 +64,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * [#645](https://github.com/binarylogic/authlogic/pull/645) -
     Add support for ruby 2.6
 
+## 4.4.3 (Unreleased)
+
+* Breaking Changes
+* None
+* Added
+* [#660](https://github.com/binarylogic/authlogic/pull/660) -
```

Comment:
@jaredbeck since we're just porting this from 4-4 to 5-0, think should we make another log entry for it? I guess in the future it might make sense to merge into the latest stable branch, and the backport to older versions 🤷‍♂️